configure/os/CONFIG_SITE.darwinCommon.darwinCommon

```diff
@@ -6,24 +6,13 @@
 # These settings are designed for users of Homebrew.
 # Users of other third-party package managers are welcome to
 # provide patches appropriate for their manager.
-ifneq (,$(wildcard /opt/homebrew))
-  # Default location on aarch64
-  HOMEBREW_DIR = /opt/homebrew
-else ifneq (,$(wildcard /usr/local/Homebrew))
-  # Default location on x86_64
-  HOMEBREW_DIR = /usr/local
-else ifneq (,$(wildcard /opt/local/include/readline))
-  # MacPorts
-  READLINE_DIR = /opt/local
-endif
+#HOMEBREW_DIR := $(shell brew --prefix 2>/dev/null)
+INCLUDES += $(HOMEBREW_DIR:%=-I%/include)
+LDFLAGS += $(HOMEBREW_DIR:%=-L%/lib)
```

Comment:
Does leaving the `HOMEBREW_DIR` line commented out put strange values into the `INCLUDES` and `LDFLAGS` variables here? Did you comment that line out because it dies unless `brew` is installed?

The point of the code you're replacing was that the user shouldn't have to edit this `CONFIG_SITE` file at all if they have Homebrew installed in either of the two standard locations as described in the comments. I don't want to undo that convenience.

Reply:
Oops, that should not have gone in. It was my test that everything works even if homebrew is not installed. I need to remove the comment.